plugins/mkt-queue-sync/src/src/Data/PluginData.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FioCz\Data;
```

Comment:
refactor: namespace

Reply:
@Piskvor namespaces of this plugin are all FioCz, because i modified the FioCz of Ubiquiti to get it working. It was not fully coded by me.
I tried to change all the namespaces but i got a lot of errors, and, as my PHP knowledge is very basic  i did the best i can. As you can read in the ReadMe file, i know that someone with more known of PHP coding can improve this code a lot.... 
But, until now, it is working great.